dashboards/gigamon-appinsights/gigamon-appinsights.json

```diff
@@ -0,0 +1,1983 @@
+            "id": "viz.table"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Gigamon SELECT count(*) FACET  bgp_nlri_prefix,bgp_nlri_prefix_len,bgp_identifier  WHERE bgp_nlri_prefix IS NOT NULL SINCE 30 DAYS AGO"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        },
+        {
+          "title": "Path Attribute Type",
+          "layout": {
+            "column": 1,
+            "row": 4,
+            "width": 4,
+            "height": 2
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.table"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "Select count(*) from Gigamon FACET bgp_path_attr_type,bgp_identifier where bgp_path_attr_type IS NOT NULL SINCE 90 days ago"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        },
+        {
+          "title": "BGP messages seen",
+          "layout": {
+            "column": 5,
+            "row": 4,
+            "width": 4,
+            "height": 4
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.bar"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Gigamon SELECT count(*) FACET  bgp_message_type,bgp_identifier SINCE 90 DAYS AGO"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        },
+        {
+          "title": "",
+          "layout": {
+            "column": 1,
+            "row": 6,
+            "width": 4,
+            "height": 2
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.table"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Gigamon SELECT count(*) FACET  bgp_error_subcode, bgp_identifier where bgp_error_subcode IS NOT NULL SINCE 90 DAYS AGO"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        }
+      ]
+    },
+    {
+      "name": "DHCP Insights",
+      "description": null,
+      "widgets": []
+    }
```

Comment:
The validation does not allow for blank widget fields, could we remove this since it doesn't have anything?
```suggestion
```